src/ResourceManager/StorageSync/Commands.StorageSync/Cmdlets/InvokeCompatibilityCheckCmdlet.cs

```diff
@@ -26,7 +26,7 @@ namespace Microsoft.Azure.Commands.StorageSync.Evaluation.Cmdlets
     using Models;
 
     [Cmdlet("Invoke", Azure.Commands.ResourceManager.Common.AzureRMConstants.AzureRMPrefix + "StorageSyncCompatibilityCheck",DefaultParameterSetName="PathBased")]
```

Comment:
@afedyashov [from the review](https://github.com/Azure/azure-powershell-cmdlet-review-pr/issues/193#issuecomment-438854797), it looked like this cmdlet was going to be renamed to be `Test-AzStorageSyncCompatibility` with an alias to `Invoke-AzStorageSyncCompatibilityCheck` -- any reason this wasn't done?

Reply:
Yes, @cormacpayne, the reason is that this PR addresses business work item below
Task 3229876: Eval Tool output to destination and format suitable for WAC
renaming commandlet came up as a side issue and we agreed to address it, but not as part of Task 3229876.
I have a mail thread discussing this with @MiYanni where I highlighted the following:

```
Just to clarify the commitment:
-	My current commitment is for the programmatic exposure of the summary, because there are partners depending on it.
-	I’m not having ETA for renaming. Renaming is lower priority for us and we are most likely postpone implementation until early 2019. 

```